index.html

```diff
@@ -565,8 +569,9 @@ <h3>Validate Algorithm</h3>
 <strong>credentialToValidate</strong>.
         </li>
         <li>
-Verify all proofs associated with the <strong>credentialToValidate</strong>.
-If a proof fails, return a validation error.
+Dereference the <code>statusListCredential</code> URL, and ensure that all
+proofs verify successfully. If the dereference fails, or if any of the proof
+verifications fail, return a validation error.
```

Comment:
Saying "return HTTP 404" (instead of just "return a validation error") doesn't make sense here because these processing rules are for consumers. This statement is telling the client / processor that is trying to dereference the URL what to do. There's no expectation that this processing software must be running on a server itself.

We could say that one possible reason for a URL dereference failure is for the consumer to receive a HTTP 404 from the server that is the authority for that URL. So, we could say that here, but what then about other dereference failures / different error codes from the server? I'd think at that point, that if the URL is an HTTPS URL, we would want to say that any 4xx/5xx error should be considered a dereferencing failure.

Reply:
+1 to handle this in another issue and get this PR merged.

Reply:
Issue https://github.com/w3c/vc-status-list-2021/issues/64 has been raised to track what HTTP responses are expected. Can you please re-review, @selfissued?